x/clp/keeper/keeper.go

```diff
@@ -97,3 +98,20 @@ func (k Keeper) GetNormalizationFactorFromAsset(ctx sdk.Context, asset types.Ass
 	}
 	return k.GetNormalizationFactor(registryEntry.Decimals)
 }
+
+func (k Keeper) GetSymmetryThreshold(ctx sdk.Context) sdk.Dec {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.SymmetryThresholdPrefix)
+	if bz == nil {
+		return sdk.NewDecWithPrec(1, 4)
+	}
+	var setThreshold types.MsgSetSymmetryThreshold
+	k.cdc.MustUnmarshal(bz, &setThreshold)
+	return setThreshold.Threshold
+}
+
+func (k Keeper) SetSymmetryThreshold(ctx sdk.Context, setThreshold *types.MsgSetSymmetryThreshold) {
+	store := ctx.KVStore(k.storeKey)
+	bz := k.cdc.MustMarshal(setThreshold)
+	store.Set(types.SymmetryThresholdPrefix, bz)
```

Comment:
@timlind should we verify the threshold value that is passed here as an input remains within a range?